inference-engine/ie_bridges/java/tests/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2020 Intel Corporation
+
+file(DOWNLOAD https://search.maven.org/remotecontent?filepath=junit/junit/4.13/junit-4.13.jar 
+                ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/junit-4.13.jar)
+file(DOWNLOAD https://search.maven.org/remotecontent?filepath=org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar 
```

Comment:
Can we use cmake to find these libraries?

Reply:
question to @likholat @dkurt . this part is not touched by me

Reply:
@ilyachur what do you mean by "use cmake to find libraries"

Reply:
Something like:
```
find_package(Java REQUIRED)
include(UseJava)
find_package(Hamcrest REQUIRED)
```
I just think that direct links is not a good solution.

Reply:
@likholat, are JUnit and Hamcrest can be installed on the system in this way?